go/test/endtoend/tabletmanager/tablegc/tablegc_test.go

```diff
@@ -326,21 +351,7 @@ func TestPurge(t *testing.T) {
 	}
 
 	time.Sleep(15 * time.Second) // purgeReentranceInterval
```

Comment:
Similar nit here. We could have a variable called `purgeReentranceInterval` so that those vars are all together and it's clear/easy to adjust the waits.

Reply:
done